instrumentation/go.mongodb.org/mongo-driver/tags.go

```diff
@@ -17,16 +17,16 @@ package mongo
 import "go.opentelemetry.io/otel/label"
 
 const (
-	TargetHostKey   = label.Key("out.host")
-	TargetPortKey   = label.Key("out.port")
-	HTTPMethodKey   = label.Key("http.method")
-	HTTPCodeKey     = label.Key("http.code")
-	HTTPURLKey      = label.Key("http.url")
-	SpanTypeKey     = label.Key("span.type")
-	ServiceNameKey  = label.Key("service.name")
-	ResourceNameKey = label.Key("resource.name")
-	ErrorKey        = label.Key("error")
-	ErrorMsgKey     = label.Key("error.msg")
+	TargetHostKey  = label.Key("out.host")
+	TargetPortKey  = label.Key("out.port")
+	HTTPMethodKey  = label.Key("http.method")
+	HTTPCodeKey    = label.Key("http.code")
+	HTTPURLKey     = label.Key("http.url")
```

Comment:
It looks like `HTTP*` keys are unused and look out of place (and http.code should be http.status_code). Should I remove them?

Reply:
Yes, if they're unused there's no point keeping them around.  There should be equivalents in the `semconv` package anyways.